### ISSUE TITLE
 Initializing libiomp5md.dll, but found libiomp5md.dll already initialized.

### ISSUE THREAD

**Issue by Abhiramisp1996** [spyder-ide/spyder#21690]
### What steps will reproduce the problem?

import cv2
from matplotlib import pyplot as plt
import numpy as np
import imutils
import easyocr
import os
# Set environment variable to avoid OpenMP duplicate library issue
os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"

# Ensure compatibility with Pillow<9.0
if not hasattr(PIL.Image, 'Resampling'):
    PIL.Image.Resampling = PIL.Image
    
    
#Read in Image, Grayscale and Blur
img = cv2.imread("frame_0000.jpg")
gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
plt.imshow(cv2.cvtColor(gray, cv2.COLOR_BGR2RGB))
#Apply filter and find edges for localization
bfilter = cv2.bilateralFilter(gray, 11, 17, 17) #Noise reduction
edged = cv2.Canny(bfilter, 30, 200) #Edge detection
plt.imshow(cv2.cvtColor(edged, cv2.COLOR_BGR2RGB))
#Find Contours and Apply Mask
keypoints = cv2.findContours(edged.copy(), cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
contours = imutils.grab_contours(keypoints)
contours = sorted(contours, key=cv2.contourArea, reverse=True)[:10]
location = None
for contour in contours:
    approx = cv2.approxPolyDP(contour, 10, True)
    if len(approx) == 4:
        location = approx
        break
print(location)

### Traceback
```python-traceback
--- Logging error ---
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\logging\__init__.py", line 1084, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\site-packages\spyder\app\start.py", line 232, in <module>
    main()
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\site-packages\spyder\app\start.py", line 228, in main
    mainwindow.main(options, args)
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\site-packages\qtconsole\manager.py", line 27, in poll
    super().poll()
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\site-packages\jupyter_client\restarter.py", line 116, in poll
    self.kernel_manager.restart_kernel(now=True, newports=newports)
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\site-packages\jupyter_client\manager.py", line 510, in restart_kernel
    self.start_kernel(**self._launch_args)
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\site-packages\jupyter_client\manager.py", line 342, in start_kernel
    self.post_start_kernel(**kw)
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\site-packages\qtconsole\manager.py", line 62, in post_start_kernel
    self.kernel_restarted.emit()
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\site-packages\spyder\plugins\ipythonconsole\widgets\shell.py", line 786, in _handle_kernel_restarted
    super(ShellWidget, self)._handle_kernel_restarted()
  File "C:\ProgramData\Anaconda3\envs\tensorflow_gpu\lib\site-packages\qtconsole\frontend_widget.py", line 530, in _handle_kernel_restarted
    self.log.warning("kernel restarted")
Message: 'kernel restarted'
Arguments: ()
```

## Versions

* Spyder version: 5.0.0 
* Python version: 3.8.8
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.22.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.1.0 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.8.1 (OK)
pylint >=1.0                  :  2.9.6 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
rtree >=0.8.3                 :  0.9.7 (OK)
setuptools >=39.0.0           :  52.0.0.post20210125 (OK)
sphinx >=0.6.6                :  4.2.0 (OK)
spyder_kernels >=2.0.1;<2.1.0 :  2.0.5 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  20.0.0 (OK)

# Optional:
cython >=0.21                 :  0.29.22 (OK)
matplotlib >=2.0.0            :  3.3.4 (OK)
numpy >=1.7                   :  1.19.2 (OK)
pandas >=1.1.1                :  1.2.4 (OK)
scipy >=0.17.0                :  1.4.1 (OK)
sympy >=0.7.3                 :  1.12 (OK)
```

**Comment by ccordoba12** [comment]
Hey @Abhiramisp1996, thanks for reporting. This problem should be fixed in our latest version available in Anaconda (5.4.3). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).

Let us know if it works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
